.github/workflows/pr_info.yml

```diff
@@ -0,0 +1,67 @@
+name: PR Info
+
+# This workflow builds the whole project once and:
+# - comments build deprecations/warnings (highlighting new ones since last tested PR)
+
+on:
+  pull_request:
+    branches:
+      - master
+      - stable
+
+jobs:
+  pr_info:
+    name: PR Info
+    runs-on: ubuntu-20.04
+    steps:
+    - name: 'Prepare sticky comment'
+      uses: marocchino/sticky-pull-request-comment@v2
```

Comment:
should we use an exact commit / version with this action or trust on them not introducing malicious content?

I think an exact commit would be more secure.

(2 occurrences of this)